tensorboard/compat/proto/plugin_pr_curve.proto

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Can we use the version of this in the plugin directory instead?

https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/pr_curve/plugin_data.proto